docs/source/recipes/color-picker-widget.md

```diff
@@ -0,0 +1,150 @@
+---
+myst:
+  html_meta:
+    "description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:title": "How to use the color picker widget"
+    "keywords": "Volto, Plone, frontend, React, blocks, forms, widget, color, picker"
+---
+
+# Color picker widget
+
+Volto comes with a color picker widget that can be used in any Volto form.
+You can combine the color picker widget with the {doc}`../blocks/block-style-wrapper` to have a powerful, yet simple way to manage color properties in your blocks.
+You can use it either in your custom block's setting's schema or enhance an existing block as follows:
+
+```js
```

Comment:
We need to use reStructuredText fallback syntax, instead of CommonMark shorthand, to get emphasize lines. In a future version of the theme, we will be able to use the shorthand.

````suggestion
```{code-block} js
````